cmake/nuttx_redefine_symbols.cmake

```diff
@@ -28,6 +28,7 @@ set(NXSYMBOLS
     bind
     calloc
     chmod
+    chdir
```

Comment:
but, why not use the same list as Makefile?

Reply:
> but, why not use the same list as Makefile?

@xiaoxiang781216 you are right, I think this is because this feature was originally prototyped in CMake and it was not thought of using preprocess.
I will try to use the original nuttx-names.in

Reply:
> but, why not use the same list as Makefile?

I changed to use the original redefine file, please review